### PR TITLE
IPF coloring added to NF plotting

### PR DESCRIPTION
**Pull request by rachelelim**
Plotting post-NF reconstruction now uses IPF colors instead of some arbitrary color scheme.

### CODE REVIEW

File: scripts/process_nf.py [hunk 1]
```diff
@@ -354,25 +354,11 @@
...
-
-        plt.figure(1)
-        plt.imshow(confidence_map[0])
-        plt.title('Bottom Layer Confidence')
 
-        plt.figure(2)
-        plt.imshow(confidence_map[-1])
-        plt.title('Bottom Layer Confidence')
+mat = Material(name=mat_name, material_file=mat_file, dmin=valWUnit(
+    'lp', 'length',  0.05, 'nm'), kev=valWUnit('kev', 'energy', beam_energy, 'keV'))
```
Comment (psavery) [inline scripts/process_nf.py:359]: Maybe this should be moved into the `if output_plot_check:` block since it is only used there?

**Review by psavery — APPROVED** [review]
This looks good to me.

@darrencpagan, did you want to look over this and/or test it as well?

**Comment by rachelelim** [comment]
I'm not sure how to revert those changes that accidentally got added on top of the IPF color change

**Comment by psavery** [comment]
The last 6 commits on this PR were merged accidentally. However, considering that they only added new files, and did not edit any existing files, we are going to just leave them.

**Pull request merged**